daemon/algod/api/server/v2/utils.go

```diff
@@ -626,6 +628,68 @@ func convertSimulationRequest(request PreEncodedSimulateRequest) simulation.Requ
 	}
 }
 
+// sortTxleaseJSONSerializableSlice lexicographically sorts a slice of TxleaseJSONSerializable
+func sortTxleaseJSONSerializableSlice(s []TxleaseJSONSerializable) {
```

Comment:
Are these actually not covered or are we relying on the devmode tests to cover?

Reply:
No longer relevant due to 2621ac8